Source/JavaScriptCore/wasm/WasmOMGIRGenerator.cpp

```diff
@@ -3269,6 +3282,21 @@ void OMGIRGenerator::emitNullCheck(Value* ref, ExceptionType exceptionType)
     });
 }
 
+bool OMGIRGenerator::emitNullCheckBeforeAccess(Value* ref, ptrdiff_t offset)
```

Comment:
Should we rename it to `canUseFaultSignalForNullCheck ` since `if (offset <= maxAcceptableOffsetForNullReference()) return true;`?